source/common/request_info/utility.cc

```diff
@@ -31,7 +31,7 @@ void ResponseFlagUtils::appendString(std::string& result, const std::string& app
 const std::string ResponseFlagUtils::toShortString(const RequestInfo& request_info) {
   std::string result;
 
-  static_assert(ResponseFlag::LastFlag == 0x1000, "A flag has been added. Fix this code.");
+  static_assert(ResponseFlag::LastFlag == 0x1200, "A flag has been added. Fix this code.");
```

Comment:
For this, and all the other asserts related to the LastFlag, we not only need to fix the value. There is accompanying code that needs to be fixed. For example here we need to add the new flag to the mapping between the enum and the string.

Any suggestions appreciated on how to make this maintenance more obvious.

Reply:
Oh. Sorry..did not realize that. Will change